internal/quickstart/container.go

```diff
@@ -162,5 +165,9 @@ func (m ContainerModel) View() string {
 		return ""
 	}
 
+	if m.gettingStarted {
+		out = "Within this guided setup flow, you'll be creating a new feature flag and,\nusing the SDK of your choice, building a small sample application to see a\nfeature flag toggle on and off in real time.\n\nLet's get started!\n" + out
+	}
+
```

Comment:
added to the container view because I wanted it above the `Step 1 of 4` bit, but let me know if we'd rather have this in the `createFlagModel` view, because that text will jump around a bit in the next step

![jump-scare](https://github.com/launchdarkly/ldcli/assets/55991524/1dd752ac-7151-49d4-b80d-cee86a65c1e3)